### PR TITLE
Lk pd 2198 make cb tag atac bam

### PR DESCRIPTION
**Pull request by ekiernan**
### Description

Added ATAC barcodes from FASTQ read names to a CB tag in the BAM. Relates to this ticket: https://broadworkbench.atlassian.net/jira/software/projects/PD/boards/167?selectedIssue=PD-2198.

----

### Checklist 
If you can answer "yes" to the following items, please add a checkmark next to the appropriate checklist item(s) **and** notify our WARP documentation team by tagging either @ekiernan or @kayleemathews in a comment on this PR.

- [ ] Did you add inputs, outputs, or tasks to a workflow?
- [ ] Did you modify, delete or move: file paths, file names, input names, output names, or task names?
- [ ] If you made a changelog update, did you update the pipeline version number?

### CODE REVIEW

**Review by nikellepetrillo — APPROVED** [review]
looks good!

File: beta-pipelines/multiome/atac.wdl [hunk 1]
```diff
@@ -315,4 +320,33 @@ task CreateFragmentFile {
...
+  >>>
+
+  output {
+    File sorted_cb_bam = "~{bam_cb_output_name}"
+  }
+
+  runtime {
+    docker: "us.gcr.io/broad-gotc-prod/samtools-bwa:1.0.0-0.7.17-1678998091"
+    disks: "local-disk ${disk_size} HDD"
+    cpu: 1
```
Comment (aawdeh) [inline beta-pipelines/multiome/atac.wdl:349]: Does CPU always need to be set to 1?

**Review by aawdeh — APPROVED** [review]
Added a small comment. But other than that, it looks good to me!

File: beta-pipelines/multiome/atac.wdl (hunk 1 above)
Comment (ekiernan) [reply beta-pipelines/multiome/atac.wdl:349]: Probably not! I'll take out

**Pull request merged**